sktime/annotation/tests/test_all_annotators.py

```diff
@@ -23,4 +24,4 @@ def test_output_type(Estimator):
     estimator.fit(*args)
     args = _make_args(estimator, "predict")
     y_pred = estimator.predict(*args)
-    assert isinstance(y_pred, pd.Series)
+    assert isinstance(y_pred, pd.Series) or isinstance(y_pred, np.ndarray)
```

Comment:
this could work as well

```suggestion
    assert isinstance(y_pred, (pd.Series, np.ndarray))
```